internal/config.go

```diff
@@ -94,3 +100,26 @@ func (c *Config) MaxSyncSetNameLength() int {
 	}
 	return maxLength
 }
+
+func substituteEnvVars(c []byte) ([]byte, error) {
+	t, err := template.New("config").Option("missingkey=error").Parse(string(c))
+	if err != nil {
+		return nil, err
+	}
+	var b bytes.Buffer
+	if err = t.Execute(&b, getEnvMap()); err != nil {
+		return nil, err
+	}
+	return b.Bytes(), nil
+}
+
+func getEnvMap() map[string]string {
+	e := os.Environ()
+	vars := make(map[string]string, len(e))
+	for _, s := range e {
+		spl := strings.SplitN(s, "=", 2)
+		k, v := spl[0], spl[1]
```

Comment:
I'm wondering if it would be worth trimming spaces. Maybe that's redundant.

Reply:
Good call. Trimming does not appear to be done by the `os` package.